### ISSUE TITLE
Can't create new event - postgres issue with gen_random_uuid

### ISSUE THREAD

**Issue by alxdean** [microsoft/azure-openai-service-proxy#172]
Just ran a fresh deployment and can't create a new event. The code change from 18 jan seems to be missing a reference?

When trying to save a new event it fails with a Npgsql.PostgresException

aoai.gen_random_uuid() does not exist

planning on running a hands-on-workshop during the swiss data science conference using Azure Open AI and this repo would have been mint! But sadly I can't get past this issue.

**Comment by alxdean** [comment]
really weird stuff going on. I redeployed using the old uuid-ossp module, but that did not work of course. not in the allowed extension list. once I dropped and recreated the add_event function it worked. (was a pain getting access as the sqladminpassword is not easily retrieved. I ended up going mixed-mode auth and managed to add my entra-id as additional admin.

so status right now: when I drop and recreate the add_event function, then the code works
